src_text/net_crypto.txt

```diff
@@ -77,15 +77,15 @@ Handshake packet:
 
 The packet id for handshake packets is 26. The cookie is a cookie obtained by sending a cookie request packet to the peer and getting a cookie response packet with a cookie in it. It may also be obtained in the handshake packet by a peer receiving a handshake packet (Other Cookie).
 
-The nonce is a nonce used to encrypt the encrypted part of the handshake packet. The encrypted part of the handshake packet is encrypted with the long term keys of both peers. This is to make sure that both peers can make sure they are really speaking to the peer they think they are speaking to.
+The nonce is a nonce used to encrypt the encrypted part of the handshake packet. The encrypted part of the handshake packet is encrypted with the long term keys of both peers. This is to prevent impersonation. 
```

Comment:
This is intended to prevent impersonation by allowing peers to verify the intended public key was used.